docs/HowTo/Interact/APIs/Authentication.md

```diff
@@ -117,10 +122,12 @@ Create the JWT token using an external tool.
 !!! important
     The JWT token must use the `RS256` algorithm 
 
-Each payload for the JWT token must contain:
+Each payload for the JWT token contains:
 
-* [JSON-RPC permissions](#json-rpc-permissions)
-* [`exp` (Expiration Time) claim](https://tools.ietf.org/html/rfc7519#section-4.1.4). 
+* [JSON-RPC permissions](#json-rpc-permissions).
+* [`exp` (Expiration Time) claim](https://tools.ietf.org/html/rfc7519#section-4.1.4).
+* Optional. The tenant's Orion public key using `privacyPublicKey`. Only used
+for multi-tenancy. 
```

Comment:
Might be worth adding a link to the multi-tenancy concept page

Reply:
Done